From 317ee4fadccc4ef1b6ff1f83e782abd3d9b7914b Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Thu, 15 Jan 2015 10:03:53 +0800
Subject: [PATCH] Makefile: clean boot.bin

When build for Atmel related boards which support SPL,
it will generate boot.bin, also clean when it when do
"make clean" operation.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5277781..a8c8230 100644
--- a/Makefile
+++ b/Makefile
@@ -1216,7 +1216,7 @@ CLEAN_FILES += u-boot.lds include/bmp_logo.h include/bmp_logo_data.h \
 
 # Directories & files removed with 'make clobber'
 CLOBBER_DIRS  += spl tpl
-CLOBBER_FILES += u-boot* MLO* SPL System.map
+CLOBBER_FILES += boot* u-boot* MLO* SPL System.map
 
 # Directories & files removed with 'make mrproper'
 MRPROPER_DIRS  += include/config include/generated          \
-- 
1.9.1

